beam/www/beam/index.ts

```diff
@@ -71,9 +71,9 @@ const routes: RouteRecordRaw[] = [
 		meta: { requiresAuth: true },
 	},
 	{
-		path: '/transfer',
-		name: 'transfer',
-		component: Transfer,
+		path: '/demand',
+		name: 'demand',
+		component: Demand,
```

Comment:
I think it may have shown up from a bad merge, but this version without the `meta` information is safe to remove